nginx/hydrodev-vb.example.org.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
@selimnairb - Correct, these are self generated certs that have nothing to do with our *.hydroshare.org certs. The intent is to provide a means for the developer to replicate (as close as possible) the production environment if they wish to. This would include nginx, uwsgi and SSL certs.

The cert will identify as hydrodev-vb.example.org, but not be recognized by any real authority.